app/assets/javascripts/discourse/components/d-editor.js.es6

```diff
@@ -636,23 +638,44 @@ export default Ember.Component.extend({
     return null;
   },
 
+  @computed
+  pastePlaceholder() {
+    return I18n.t('pasting');
+  },
+
   paste(e) {
```

Comment:
Generally speaking I find this method hard to follow, the meaning of `handled` is not always clear to me, and part of the code could probably be more explicit.

For example, could we replace this by a name, fully or partially ?
```
(handled || (types.some(t => t === "Files") && !plainText))
```